docs/book/src/clusteropenstack/configuration.md

```diff
@@ -327,6 +327,13 @@ A floating IP is created and associated to the bastion host automatically, but y
 
 If `managedSecurityGroups: true`, security group rule opening 22/tcp is added to security groups for bastion, controller, and worker nodes respectively. Otherwise, you have to add `securityGroups` to the `bastion` in `OpenStackCluster` spec and `OpenStackMachineTemplate` spec template respectively.
 
+If PortSecurity disable for all ports either by setting `OpenStackCluster.spec.disablePortSecurity: true` or for each port individually, `managedSecurityGroups: false` is the only valid option. The following is not a valid coonfiguration and is not supported.
```

Comment:
This is in "Accessing nodes through the bastion host via SSH" section, not security group section. How about moving to [Ports](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/book/src/clusteropenstack/configuration.md#ports)?

Reply:
Thanks, moved